operations/patch_util.go

```diff
@@ -421,6 +426,36 @@ func gitCommitMessages(base, ref string) (string, error) {
 	return gitCmd("log", args...)
 }
 
+// assumes base includes @{upstream}
+func gitLastCommitMessage() (string, error) {
+	args := []string{"HEAD", "--no-show-signature", "--pretty=format:%s", "-n 1"}
+	return gitCmd("log", args...)
+}
+
+func gitBranch() (string, error) {
+	args := []string{"--abbrev-ref", "HEAD"}
+	return gitCmd("rev-parse", args...)
+}
+
+func getDefaultDescription() string {
+	desc, err := gitLastCommitMessage()
+	if err != nil {
+		grip.Debug("Couldn't create patch description using commit messages.")
```

Comment:
I don't think this `grip.Debug` statement of the one below are useful, since they don't include an error message. I think we should either ignore the error, e.g.,
```
desc, _ := gitlastCommitMessage
```
or we should include the error message in the log statement. Currently there will be a problem message printed to the user, but there will be no easy way for us to debug it when they ask us about it.

Reply:
I don't really want to swallow the error since we won't likely know if it's not working otherwise.